### ISSUE TITLE
Clicking output control throws error in JS console

### ISSUE THREAD

**Issue by novwhisky** [jaedb/Iris#806]
### Iris version

3.61.1

### Operating system(s) affected

- [ ] Windows
- [X] MacOS
- [X] iOS
- [ ] Android
- [ ] Linux
- [ ] Other

### Browser(s) affected

- [X] Firefox
- [X] Chrome
- [ ] Edge
- [ ] Other

### What happened?

After upgrading to snapserver 0.26.0 from a mistaken Debian repo install, clicking on the outputs speaker icon causes the Iris UI to go blank and a TypeError to be thrown in the web console when Snapcast is enabled. Refreshing the page will bring the controls back, but AFAICT there's no way to configure outputs as had worked previously. Is this a JSON RPC version issue?

Here's a screen recording of the error

https://user-images.githubusercontent.com/1056369/152872827-5e25bde7-c753-49b1-9ec5-8aa4ca70e22d.mov

### Logs

```shell
TypeError: Cannot read properties of undefined (reading 'scheme')
    at A (app.min.js?v=1643875123:79:192392)
    at Wa (app.min.js?v=1643875123:95:57930)
    at bs (app.min.js?v=1643875123:95:104169)
    at uc (app.min.js?v=1643875123:95:96717)
    at sc (app.min.js?v=1643875123:95:96642)
    at Zs (app.min.js?v=1643875123:95:93672)
    at app.min.js?v=1643875123:95:45314
    at t.unstable_runWithPriority (app.min.js?v=1643875123:103:3844)
    at Hi (app.min.js?v=1643875123:95:45023)
    at Gi (app.min.js?v=1643875123:95:45259)
```

**Comment by bendschs** [comment]
i get a black screen when i tip on outputs on latest IOS in safari.

**Comment by novwhisky** [comment]
I was able to go deeper into debugging Iris by going to "Settings", clicking "<> Debug" in the top-right corner, enabling "Test mode", checking "Log actions" and reloading Iris. That turned on verbose console output and allowed me to better inspect the React/Redux componentry.

Looks like this was an issue with orphaned snapcast client entries, possibly from prior to upgrading snapserver. The web console will log a `persist/REHYDRATE` action object with `key: 'snapcast'`. If you expand `payload.clients` there's probably a lot of random hex strings where `connected: false`. I noted down a list of all these IDs and used the [JSON-RPC to delete each them](https://github.com/mjaggard/snapcast/blob/master/doc/json_rpc_api/v2_0_0.md#serverdeleteclient) one-by-one, i.e.

```zsh
$ telnet localhost 1705

> {"id":2,"jsonrpc":"2.0","method":"Server.DeleteClient","params":{"id":"target-id-to-delete-1"}}
> {"id":2,"jsonrpc":"2.0","method":"Server.DeleteClient","params":{"id":"target-id-to-delete-2"}}
> {"id":2,"jsonrpc":"2.0","method":"Server.DeleteClient","params":{"id":"target-id-to-delete-3"}}
```

Successful requests will respond with a large client object. I also added a meaningful [`--hostID` name](https://github.com/badaix/snapcast/blob/master/client/snapclient.1#L34-L35) argument to all of my snapclients in `/etc/default/snapclient` (NOT the snapcast server) and restarted so I can distingush them more easily.

Reloading Iris after that I can now control Snapcast by clicking the speaker icon again. For @jaedb I would recommend using [`componentDidCatch`](https://reactjs.org/docs/react-component.html#componentdidcatch) around the [OutputControl](https://github.com/jaedb/Iris/blob/master/src/js/components/Fields/OutputControl.js) component to prevent the entire app from unmounting in scenarios like this.
